pytrip/utils/spc2pdf.py

```diff
@@ -159,7 +178,10 @@ def main(args=sys.argv[1:]):
                         energy_steps = data.energy[m3]
                         fluence = data.fluence[m3]
                         if np.any(m3):
-                            ax.plot(energy_steps, fluence, label="Z = {:d}, depth = {:3.3f} [cm]".format(z, depth_step))
+                            ax.plot(energy_steps, fluence,
+                                    linestyle=linestyle,
+                                    marker=marker,
+                                    label="Z = {:d}, depth = {:3.3f} [cm]".format(z, depth_step))
```

Comment:
do we need to repeat the depth here, if it is already set in the title? We can save some legend space.

Reply:
There are two depths: one requested by user, which goes into title, and another one for which spectrum is shown (from the one available in SPC file) which goes into legend. They may differ.  See for example:

![image](https://user-images.githubusercontent.com/7374733/39528366-345c9442-4e24-11e8-8e9f-919e8833c205.png)

Reply:
So there are two different kinds of depth. Need better naming scheme then. If you show that plot above to anyone, they will be confused..

Reply:
What do you think about following:
![image](https://user-images.githubusercontent.com/7374733/39533901-8bf05692-4e30-11e8-93d3-d8969d9b23cd.png)

?

Reply:
Fixed in c31aa24